src/export_star_transformer.ts

```diff
@@ -0,0 +1,59 @@
+import * as ts from "typescript";
+
+/**
+ * Closure compiler cannot handle:
+ *
+ * ```
+ * export * as namespace from "./module";
+ * ```
+ *
+ * This transformer changes star namespace exports to the following:
+ *
+ * ```
+ * import * as namespace from "./module";
+ * export {namespace};
```

Comment:
Is it legal for `namespace` to also be an identifier in the file scope? 
Should we be renaming it somthing like __tsickle_namespace to avoid conflicts?

Reply:
Yeah, it's legal, but also I think we have to leave it alone because it's valid for something in the current file to use.

```ts
export * as ns from "./module";
ns.someMethod();
```
is valid, so renaming the `ns` would break that (or at least require us to also rename variables throughout).

So I guess what I'm saying is that it's unlikely that there would be a conflict given we're keeping the names the same as what they were prior to the transformation.